### PR TITLE
Add healthcheck

### PR DESCRIPTION
**Pull request by jeremyhiggs**
The change adds a check on the status of the Omada controller in the docker container.

### CODE REVIEW

File: Dockerfile.v3.2.x [hunk 1]
```diff
@@ -1,4 +1,5 @@
 FROM ubuntu:18.04
+HEALTHCHECK --start-period=15m CMD wget --quiet --tries=1 --no-check-certificate http://127.0.0.1:8088 || exit 1
```
Comment (mbentley) [inline Dockerfile.v3.2.x]: Just curious why a start period of 15m; that seems a bit excessive, no?  I was able to start and get to the point of receiving `Omada Controller started` in about 1 minute on an embedded box that has an `AMD G-T40E` which has a max clock speed of 1 GHz.  I would think that maybe 2-3 minutes; or 5 minutes if we are looking for a huge cushion might be good unless you're looking at a different use case.  Thoughts?

**Comment by jeremyhiggs** [comment]
Ah, I copied this from a previous Dockerfile I had set up for running Omada
on a Raspberry Pi 3. Eventually gave up on that as it was slowww and
hobbled by RAM. I think 5m is a fair time period for any desktop-class
machine.

On 29 February 2020 at 4:32:06 PM, Matt Bentley (notifications@github.com)
wrote:

*@mbentley* commented on this pull request.
------------------------------

In Dockerfile.v3.2.x
<https://github.com/mbentley/docker-omada-controller/pull/22#discussion_r386022029>
:

> @@ -1,4 +1,5 @@
 FROM ubuntu:18.04
+HEALTHCHECK --start-period=15m CMD wget --quiet --tries=1
--no-check-certificate http://127.0.0.1:8088 || exit 1

Just curious why a start period of 15m; that seems a bit excessive, no? I
was able to start and get to the point of receiving Omada Controller started
in about 1 minute on an embedded box that has an AMD G-T40E which has a max
clock speed of 1 GHz. I would think that maybe 2-3 minutes; or 5 minutes if
we are looking for a huge cushion might be good unless you're looking at a
different use case. Thoughts?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/mbentley/docker-omada-controller/pull/22?email_source=notifications&email_token=AABMPUGI7DBR62ICATDI2A3RFDY3NA5CNFSM4K6QVZKKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCXONFAA#pullrequestreview-366793344>,
or unsubscribe

**Comment by mbentley** [comment]
Makes sense.  Thanks for the contribution!

**Comment by mbentley** [comment]
Sorry, I never received notification of the changes being updated.

**Pull request merged**